src/pretix/plugins/paypal/payment.py

```diff
@@ -531,3 +532,16 @@ def shred_payment_info(self, obj):
             le.data = json.dumps(d)
             le.shredded = True
             le.save(update_fields=['data', 'shredded'])
+
+    def render_invoice_text(self, order: Order, payment: OrderPayment) -> str:
+        if order.status == Order.STATUS_PAID:
+            if payment.info_data.get('id', None):
+                return '{}\r\n{}: {}'.format(
+                    _('The payment for this invoice has already been received.'),
+                    _('PayPal payment ID'),
+                    payment.info_data['id']
```

Comment:
We should *also* display the sale ID that, if it exists, is stored at `payment.info_data['transactions'][0]['related_resources'][0]['sale']['id']`. The ``id`` value is something that seems to be specific to the REST API, so it does not help when trying to find a transaction in the consumer or merchant portal.